### PR TITLE
Add Omega Ruby Game

### LINKED ISSUE #151: Missing Game Index
## Expected Behavior

When you click a pokemon and look at the more information section, it should show you the games they are in or none. 

## Actual/Current Behavior

If you clicked on a pokemon that was in Omega Ruby it would crash. 

## Steps to Reproduce the Problem

  1. Click Groudon Primal
  2. Click more information button in sidebar

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Added Omega Ruby to the function that prints the images of the games out

## Purpose
Keeps the app from crashing when a user clicks on a Pokemon from that game

## Approach
Import an image and add it to the function that prints out the games. 

Closes #151

### CODE REVIEW

**Review by gonzalespaulb — APPROVED** [review]

File: pokeapp/src/components/Sidebar.js [hunk 1]
```diff
@@ -39,7 +39,9 @@ const SideBar = ({
   const [showPokemon, setShowPokemon] = useState(false);
   const [pokeNameList, setPokeNameList] = useState([]);
 
-  const [videoGame, setVideoGame] = useState(`No Game Selected`);
+  console.log(selectedPokemon)
```
Comment (Lap343) [inline pokeapp/src/components/Sidebar.js]: Is this console.log still needed?

Comment (krferrell) [reply pokeapp/src/components/Sidebar.js]: Thanks! It is gone

**Review by Lap343 — APPROVED** [review]
I'd say after that console.log you're code looks fine

**Review by JayDampitan — APPROVED** [review]
Nice fix!

**Pull request merged**